conf/application-example.conf

```diff
@@ -148,6 +148,21 @@ services {
     aliases = [ ]
 
     # You must provide at least one of `indexes` or `aliases` for anything to show up in this service.
+
+    # Retry configuration defaults. You can leave this section out entirely to use these.
```

Comment:
add a comment it does exponential back-off

Reply:
Already there as `# Wait time will double with every subsequent failure.`